engine/src/flutter/lib/web_ui/lib/src/engine/scene_painting.dart

```diff
@@ -19,6 +19,14 @@ abstract class ScenePicture implements ui.Picture {
   ui.Rect get cullRect;
 }
 
+abstract class ScenePictureRecorder implements ui.PictureRecorder {
```

Comment:
Our existing convention has been to prefix with `Engine`, e.g. `EnginePictureRecorder`, but I don't feel strongly about it.